clinical/src/test/java/com/hartwig/actin/clinical/curation/CurationModelTest.java

```diff
@@ -48,15 +45,15 @@ public class CurationModelTest {
 
     private static final String CURATION_DIRECTORY = Resources.getResource("curation").getPath();
 
+    private CurationModel model = TestCurationFactory.createProperTestCurationModel();
```

Comment:
Make final static?

Reply:
I'll make it final, but as above I think it's better to remain non-static.